e2e/test/scenarios/question/reproductions/22285-schema-picker.cy.spec.js

```diff
@@ -1,35 +0,0 @@
-import { restore, startNewQuestion, popover } from "e2e/support/helpers";
-
-describe("issue 22285", () => {
-  beforeEach(() => {
-    restore();
-    cy.signInAsAdmin();
-
-    cy.intercept("GET", "/api/database").as("fetchDatabases");
-
-    cy.intercept("GET", "/api/database/*/schemas", {
-      body: ["PUBLIC", "FAKE SCHEMA"],
-    });
-  });
-
-  it("should not clean DB schemas list in the data selector (metabase#22285)", () => {
```

Comment:
This test/bug was specific to the old popover, where you could navigate back and forth.
I think it does not makes sense to keep it.